errata_tool/tests/test_cdn_repos.py

```diff
@@ -0,0 +1,95 @@
+import pprint
+
+
+arches = [
+    'ppc64le',
+    's390x',
+    'x86_64',
+]
+
+content_types = [
+    ('debug-rpms',  'Debuginfo'),
+    ('rpms',        'Binary'),
+    ('source-rpms', 'Source'),
+]
+
+
+def repo_sets(repo):
+    # Helper function to group RPM repos for easier looping
```

Comment:
The cdn repos in this unit test happen to be in groups of three (RPM, Source RPM, and Debuginfo for each supported architecture), which is another accidental reliance on the particular structure of the JSON. I don't believe it'd be easy to generalize (for example, doesn't really work for container repos which are multiarch and primarily are distributed as Docker repos), and if anything relies on a convention that just happens to be common in Red Hat product configuration. This function only exists to remove some duplication.

It could be refactored to use a flatter structure like for the variant tests:

https://github.com/red-hat-storage/errata-tool/blob/master/errata_tool/tests/test_variants.py#L5

Reply:
I've changed repo_sets() a little. Does it need further clarification?

Reply:
Yeah, this looks clearer to me. Thank you.